includes/definitions/ericsson-ipos.yaml

```diff
@@ -7,7 +7,6 @@ mib_dir: ericsson
 over:
     - { graph: device_bits, text: 'Device Traffic' }
     - { graph: device_processor, text: 'CPU Usage' }
-    - { graph: device_mempool, text: 'Memory Usage' }
```

Comment:
May as well swap this for device_storage seeing as you've added support for that.

Rest looks good though.

Reply:
Done!